### ISSUE TITLE
Up/Down keys broken on touch-enabled browsers

### ISSUE THREAD

**Issue by smilingthax** [freeCodeCamp/devdocs#112]
My Desktop PC is equipped with a touchscreen; my browser has
$.isTouchScreen() === true.
The keyboard navigation, namely Enter/Up/Down, does not work then, forcing me to use the mouse. 

When hot-patching $.isTouchScreen to always return false, the issue can be (temporarily) worked around.
